website/ref/language.md

```diff
@@ -2392,18 +2393,27 @@ itself or defined by the user.
 
 ### Importing modules with `use`
 
-Modules are imported using the `use` special command, which accepts a **module
-spec** and an optional alias:
+Modules are imported using the `use` special command. It requires a module spec
+and an optional alias:
 
 ```elvish
 use $spec $alias?
 ```
 
-Both the module spec and the alias must appear as a single string literal.
+The module spec and the alias must both be a [string literal](#string).
 
-The module spec specifies which module to import, and the alias, if given,
-specifies the namespace to import the module under. By default, the namespace is
-derived from the module spec, by taking the part after the last slash.
+The module spec specifies which module to import. The alias, if given, specifies
+the namespace to import the module under. By default, the namespace is derived
+from the module spec by taking the part after the last slash.
+
+Module specs fall into two categories:
```

Comment:
This description leaves out the important case of modules in module search directories.

Reply:
D'oh! Of course. I'll add an item for that case.